app/services/support_interface/interview_changes_export.rb

```diff
@@ -0,0 +1,100 @@
+module SupportInterface
+  class InterviewChangesExport
+    def data_for_export
+      interview_audits.find_each(batch_size: 100).map do |audit|
+        row_for_audit(audit)
+      end
+    end
+
+    def row_for_audit(audit)
+      interview = audit.auditable
+      application_choice = interview.application_choice
+      {
+        audit_id: audit.id,
+        audit_created_at: audit.created_at,
+        audit_type: audit.action,
+        interview_id: interview.id,
+        candidate_id: application_choice.candidate.id,
+        application_form_id: application_choice.application_form.id,
+        provider_code: application_choice.provider.code,
+        provider_user: audit_user(audit),
+        interview_status: interview_status(interview, audit),
+        interview_preferences: application_choice.application_form.interview_preferences,
+        application_submitted_at: application_choice.application_form.submitted_at,
+        course_code: application_choice.course.code,
+        course_location: application_choice.site.name_and_code,
+      }.merge(interview_change(audit))
+    end
+
+  private
+
+    def interview_audits
+      Audited::Audit.where(auditable_type: 'Interview').includes(
+        auditable: {
+          application_choice: %i[application_form course site candidate],
```

Comment:
Could you also add `user: :provider_user` to the includes and also `provider` to `application_choice`? I think this way we cover all n+1s